osu.Game.Rulesets.Mania/Mods/ManiaModNoDrain.cs

```diff
@@ -0,0 +1,30 @@
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
+
+
+
+namespace osu.Game.Rulesets.Mania.Mods
+{
+    public class ManiaModNoDrain : Mod, IApplicableToDifficulty
+    {
+        public override string Name => "No Drain";
+
+        public override string Acronym => "ND";
+
+        public override double ScoreMultiplier => 0.5;
+
+        public override string Description => "No drain for mania!";
+
+        public override ModType Type => ModType.DifficultyReduction;
+        public virtual void ReadFromDifficulty(BeatmapDifficulty difficulty)
+        {
+        }
+
+        public virtual void ApplyToDifficulty(BeatmapDifficulty difficulty)
+        {
+            difficulty.DrainRate *= 0;
```

Comment:
Difficulty Adjust mod could be used for this if you set HP drain rate to 0, so I'm not sure why this would need a whole new mod

Reply:
Is a very good point.

Reply:
While not out yet, the new mod select designs will make it more obvious that difficulty setting adjustment is possible through mods. I believe its the purpose of the Difficulty Adjust mod to begin with.